internal/pkg/flink/components/interactive_input.go

```diff
@@ -34,6 +35,6 @@ func PrintWelcomeHeader() {
 	output.Printf("To exit, press Ctrl-Q or type \"exit\". \n\n")
 
 	// Print shortcuts
-	output.Printf("\033[0m%s \033[0;36m%s \033[0m", "[Ctrl-Q]", "Quit")
-	output.Printf("\033[0m%s \033[0;36m%s \033[0m \n", "[Ctrl-S]", "Toggle Smart Completion")
+	output.Printf("[Ctrl-Q] " + color.CyanString("Quit "))
+	output.Printf("[Ctrl-S] " + color.CyanString("Toggle Smart Completion \n"))
```

Comment:
It feels weird to be coloring whitespace, and you've got a weird trailing space. Format strings are probably the way to go here.
```suggestion
	output.Println("[Ctrl-Q] %s [Ctrl-S] %s", color.CyanString("Quit"), color.CyanString("Toggle Smart Completion"))
```

Reply:
Done :) You prob mean printf and not println though. I got an error/warning when using println with the formatting options.

Reply:
![image](https://github.com/confluentinc/cli/assets/11739405/1c69cf7a-ca51-4628-85db-0f1b0d3865da)

Reply:
Oops, my bad!

Reply:
What's up with the trailing space though?